user-management/readme.md

```diff
@@ -0,0 +1,62 @@
+# User Management API Client Sample
+
+This directory contains scripts for user management tasks.
+
+## Prerequisites
+
+Before using the User Management API Client Sample, ensure the following prerequisites are met:
+
+1. **Install Requirements:**
+    - Refer to the [api-client-example README](../readme.md#prerequisites) for instructions on installing the necessary Python packages and dependencies.
+
+2. **Get Required Variables:**
+    - Obtain the following variables from your self-hosted environment or API provider:
+        - `base_url`: The base URL of the User Management API endpoint.
+        - `client_id`: The client ID for authentication.
+        - `client_secret`: The client secret for authentication.
+
+    Make sure to follow the instructions provided in the [api-client-example README](../readme.md#use-cases) to obtain and configure these variables.
+
+## Usage Limitations
+
+- This script is specifically designed for self-hosted environments.
+- Superadmin privileges are required to execute this script.
```

Comment:
let's mention to contact Datasaur support to set superadmin privileges
Tapi, karena ini public repo, kayanya jangan mention email address nya ?
Something like this maybe?:
```
- Superadmin privileges are required to execute this script. Please refer to your installation documents or contact Datasaur support for specific steps on how-to set the privileges to your account 
```

^ feel free to refine ya, ini coret-coret aja tadi

Reply:
udpated https://github.com/datasaur-ai/api-client-example/pull/22/commits/3fc2b93cc613c235eb44af7acedeaa869fd0815f